Makefile

```diff
@@ -2,7 +2,7 @@
 
 #Unit tests
 test:
-	go test -v -race -short -timeout=60s ./...
+	go test -v -race -count=1 -timeout=60s ./...
```

Comment:
why to disable cache

Reply:
Because in some case we change in code but the tests results returns from the cache instead of running tests.